package.json

```diff
@@ -49,23 +54,26 @@
   },
   "devDependencies": {
     "babel-core": "*",
-    "babel-loader": "*",
+    "babel-loader": "^6.2.7",
     "babel-polyfill": "*",
-    "babel-preset-es2015": "6.14.0",
+    "babel-preset-es2015": "^6.14.0",
+    "babel-preset-react": "^6.16.0",
+    "babel-preset-stage-1": "^6.16.0",
     "clean-css": "*",
-    "css-loader": "*",
+    "css-loader": "^0.26.0",
     "eslint": ">=3",
     "eslint-plugin-jsdoc": "*",
     "eslint-plugin-react": "*",
     "eslint-plugin-react-native": "*",
     "expose-loader": "*",
-    "file-loader": "*",
+    "file-loader": "^0.9.0",
```

Comment:
I need to know how important it is to this PR that we mention versions. I know that there are generally pros to mentioning minimal versions but I'm not sure that I want to address these (possibly) best practices in the scope of this PR. If you feel strongly about them but they are not essential to this PR, let's please have a separate PR for them.